src/vmm/src/devices/virtio/queue.rs

```diff
@@ -686,26 +686,6 @@ impl Queue {
         mem.write_obj(avail_event, avail_event_addr).unwrap();
     }
 
-    /// Helper method that writes to the `idx` field of the used ring.
-    #[inline(always)]
-    fn set_used_ring_idx<M: GuestMemory>(&mut self, next_used: u16, mem: &M) {
-        debug_assert!(self.is_valid(mem));
```

Comment:
can we keep these `debug_assert!`s in the new functions?

Reply:
new functions do not have access to guest memory object